code/modules/mob/living/sneeze.dm

```diff
@@ -0,0 +1,49 @@
+/// Launch a sneeze that can infect with a disease
+/mob/living/proc/infectious_sneeze(datum/disease/disease, force, range = 4, charge_time = 0.5 SECONDS, obj/projectile/sneezoid = /obj/projectile/sneeze)
+	sneeze(range, charge_time, sneezoid, on_sneeze_hit_callback = CALLBACK(src, PROC_REF(try_sneeze_infect), disease, force))
+
+/// Try and infect following a sneeze hit. force to always infect
+/mob/living/proc/try_sneeze_infect(datum/disease/disease, force, mob/living/target)
+	target.AirborneContractDisease(disease, force)
```

Comment:
My gut says it's preferable to just pass a list of diseases to the sneeze that are spread rather than using a callback

Reply:
I kinda wanna leave it open-ended and easily moddable, so instead of infecting I can make someone explode or something

Reply:
Currently if you get cured while the projectile is traveling the sneeze absolves itself of infectiousness which is a realism fail

Reply:
It now passes a copy of the instance, so reality is restored